images/kernel/Makefile

```diff
@@ -1,11 +1,22 @@
-KERNEL_VERSION ?= 4.19.47
-KERNEL_EXTRA ?= 
+# Check https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/refs/ for updates
+KERNEL_VERSIONS ?= 4.14.166 4.19.97 5.4.13
```

Comment:
Noting that these will not overwrite the old images @ https://hub.docker.com/r/weaveworks/ignite-kernel/tags since they are all version bumps :+1:

Reply:
Apologies in advance ... I'm just getting my head around Ignite - by spec'ing and writing a Cheff Cookbook to install `ignite and ignited`:
Is there a Linux kernel dependency to take into consideration when installing Ignite.

Reply:
@taqtiqa-mark these kernel versions are for the VM guests that ignite runs. They're distributed as OCI images so you don't have to worry about installing them.

For installing ignite, you need a linux host capable of meeting Firecrackers minimum runtime dependencies.
Currently, that is a host with Linux Kernel 4.14 or newer and KVM enabled:
- https://github.com/firecracker-microvm/firecracker/blob/master/FAQ.md#what-operating-systems-are-supported-by-firecracker

Please see our documented requirements:
- https://ignite.readthedocs.io/en/stable/dependencies.html#requirements-and-dependencies